libs/faad2/Makefile

```diff
@@ -68,14 +60,14 @@ endef
 TARGET_CFLAGS += $(FPIC)
 CONFIGURE_ARGS+= --without-xmms
 
-# Use fixed point math, if we're using soft floats, and the user didn't
-# specifically ask to use floats anyways.
-#ifeq ($(CONFIG_SOFT_FLOAT)-$(CONFIG_FAAD2_ALWAYS_USE_FLOATING_POINT),y-)
-TARGET_CFLAGS+= -DFIXED_POINT
-CONFIGURE_ARGS+= -nfp
-#endif
+ifeq ($(CONFIG_SOFT_FLOAT),y)
+	TARGET_CFLAGS+= -DFIXED_POINT
+	CONFIGURE_ARGS+= -nfp
+endif
 
-CONFIGURE_VARS+= CPPFLAGS="-fno-builtin-cos -fno-builtin-sin -fno-builtin-log"
+ifeq ($(CONFIG_USE_UCLIBC),y)
```

Comment:
And...  MUSL is used for everything but ARM, right?  What happens when that changes?